packages/core/src/components/TextField/Styled/InnerWrapper/outlined.styled.tsx

```diff
@@ -22,20 +22,27 @@ export const outlinedStyle = ({ theme, outlined, disabled, isErrorPresent }: Inn
             pointer-events: none;
             background-color: transparent;
             transition: all 100ms ease-out;
-            border-radius: ${theme.spacing.S1};
+            border-radius: ${outlined.default.borderRadius};
+            border-width: ${outlined.default.borderRadius};
             border: 0.1rem solid ${outlined.default.borderColor};
         }
         &:hover {
             box-shadow: ${!disabled && `0px 0.2rem 0.8rem ${rgba(outlined.hover.shadowColor, 0.2)}`};
         }
         &:focus-within {
-            box-shadow: ${!disabled && `0px 0.2rem 0.8rem ${rgba(outlined[isErrorPresent ? 'error' : 'active'].shadowColor, 0.2)}`};
+            border-radius: ${outlined.active.borderRadius};
+            box-shadow: 0 0.2rem 0.8rem ${rgba(outlined[isErrorPresent ? 'error' : 'active'].shadowColor, 0.2)};
```

Comment:
We need to check `disabled` also over here because we don't want to show to shadow if the field is disabled.